src/Our.Umbraco.Ditto/Models/Archetype/Abstract/ArchetypeModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Our.Umbraco.Ditto.Models.Archetype.Abstract
+{
+    public class ArchetypeModel
```

Comment:
:warning: As mentioned before, I have concerns about Ditto (core) being too aware of Archetype - especially with being responsible for deserializing it's JSON, and bypassing it's PropertyValueConverter.